src/kili/presentation/client/project.py

```diff
@@ -426,5 +431,6 @@ def count_projects(
                 updated_at_gte=updated_at_gte,
                 updated_at_lte=updated_at_lte,
                 archived=archived,
+                deleted=deleted,
```

Comment:
Add a quick description in the documentation above to explain what this field is doing

Reply:
Done ! ✅